### ISSUE TITLE
TopoJSON isn't cool. You know what's cool? Protobuf.

### ISSUE THREAD

**Issue by wboykinm** [faradayio/tilesplash#2]
Without the faintest idea of what's required, let's look into a .pbf endpoint option. Looks like [it may be the future](https://github.com/mapbox/mapbox-gl-js/pull/697#issuecomment-52379656)

**Comment by meule** [comment]
@devTristan Do you have any ideas how to implement Mapbox MVT format?

**Comment by devTristan** [comment]
@meule I do, as a matter of fact. I added MVT support, but I was getting mysterious hanging connection problems, so I reverted it in https://github.com/faradayio/tilesplash/commit/eea056d21ec102bee45ee440abdf4caf3fb5ca89. Those problems may not have been caused by the MVT support at all, I'm not quite sure. If that commit can be un-reverted without issue, it should be. If you want to, test it and tell me what you discover.
